kebechet/utils.py

```diff
@@ -91,13 +91,13 @@ def cloned_repo(manager: "ManagerBase", branch=None, **clone_kwargs):
 
     if _CLONE_DIRECTORY is not None:
         with cwd(_CLONE_DIRECTORY):
-            if os.path.isdir(os.path.join(_CLONE_DIRECTORY, ".git")):
-                repo = git.Repo(_CLONE_DIRECTORY)
+            if os.path.isdir(os.path.join(".", ".git")):
```

Comment:
are we no longer using the `_clone_directory` ?

Reply:
I came across a small issue if _CLONE_DIRECTORY refers to a local dir. Because we `cwd(_CLONE_DIRECTORY)` this is effectively the same

Reply:
I came across a small issue if _CLONE_DIRECTORY refers to a local dir. Because we `cwd(_CLONE_DIRECTORY)` this is effectively the same